Allow configuring the log level of the retry give-up message (#7567)
scrapy/scrapy: docs/topics/downloader-middleware.rst

exception propagation, see
:meth:`~scrapy.downloadermiddlewares.DownloaderMiddleware.process_exception`.

.. setting:: RETRY_GIVE_UP_LOG_LEVEL

RETRY_GIVE_UP_LOG_LEVEL

> ^^^^^^^^^^^^^^^^^^^^^^^^

Reviewer: ```suggestion
^^^^^^^^^^^^^^^^^^^^^^^
```

exception propagation, see
:meth:`~scrapy.downloadermiddlewares.DownloaderMiddleware.process_exception`.

.. setting:: RETRY_GIVE_UP_LOG_LEVEL

RETRY_GIVE_UP_LOG_LEVEL
^^^^^^^^^^^^^^^^^^^^^^^^

Default: ``"ERROR"``

:ref:`Logging level <levels>` used for the message that is logged when a
request exhausts its retries and retrying is given up. It may be a level name
(e.g. ``"WARNING"``) or a level number (e.g. ``logging.WARNING``).

Lower it (e.g. to ``"WARNING"``) if giving up on a request should not count
towards the ``log_count/ERROR`` stat.

The logging level can also be specified per-request using

> :reqmeta:`give_up_log_level` attribute of :attr:`Request.meta <scrapy.Request.meta>`.

Reviewer: ```suggestion
The logging level can also be specified per-request using the
:reqmeta:`give_up_log_level` :attr:`Request.meta <scrapy.Request.meta>` key.
```

> When initialized, the :reqmeta:`give_up_log_level` meta key takes higher

Reviewer: ```suggestion
When set, the :reqmeta:`give_up_log_level` meta key takes higher
```

Or "defined". Initialized sounds weird to me in this context.
